### PR TITLE
[Arista]: Change the TH5 Yaml settings to use 8 UC and 4 MC queues

### PR DESCRIPTION
**Pull request by Pterosaur**
#### Why I did it
The current YAML file for TH5 should be used only with 8 UC and 4 MC queues.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Change the TH5 Yaml settings to use 8 UC and 4 MC queues

#### How to verify it

Local ixia test is passing in lab.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by lguohan** [comment]
@kperumalbfn , what is this PR about? what is current # of UC and # MC queue configured?

**Comment by Pterosaur** [comment]
We discussed this internally, this PR is not required as these changes can cause flexcounter issue.

File: device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/th5-a7060x6-64pe.config.bcm [hunk 1]
```diff
@@ -2622,7 +2622,7 @@ device:
         TM_PORT_UC_Q_TO_SERVICE_POOL:
             ?
                 PORT_ID: [[1,8], [11,18], [21,29], [33,40], [43,51], [55,62], [65,73], [76,84], [87,95], [99,106], [109,117], [121,128], [131,139], [143,150], [153,161], [165,172], [175,183], [187,194], [197,205], [209,216], [219,227], [231,238], [241,249], [253,260], [263,271], [274,282], [285,293], [297,304], [307,315], [319,326], [329,337], [341,348], 351]
-                TM_UC_Q_ID: [0,1,2,5,6,8,9]
+                TM_UC_Q_ID: [0,1,2,5,6]
```
Comment (r12f) [inline device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/th5-a7060x6-64pe.config.bcm:2625]: when searching the BCM file, I am still seeing some UC Q settings on queue 8 and 9. they are removed in the first commit, but somehow removed in the second one:

E.g.:

```
                PORT_ID: [[1,8], [11,18], [22,29], [33,40], [44,51], [55,62], [66,73], [77,84], [88,95], [99,106], [110,117], [121,128], [132,139], [143,150], [154,161], [165,172]]
                TM_UC_Q_ID: [8,9]
            :
                MIN_GUARANTEE_CELLS: 0
                SHARED_LIMITS: 1
                DYNAMIC_SHARED_LIMITS: 1
                SHARED_LIMIT_CELLS_STATIC: 0
                SHARED_LIMIT_DYNAMIC: ALPHA_1
                DYNAMIC_GROUP: MID_PRI_GROUP
                RESUME_OFFSET_CELLS: 2
                COLOR_SPECIFIC_LIMITS: 0
                COLOR_SPECIFIC_DYNAMIC_LIMITS: 0
                YELLOW_LIMIT_CELLS_STATIC: 0
                YELLOW_LIMIT_DYNAMIC: PERCENTAGE_750
                RED_LIMIT_CELLS_STATIC: 0
                RED_LIMIT_DYNAMIC: PERCENTAGE_625
            ?
```

Comment (kperumalbfn) [reply device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/th5-a7060x6-64pe.config.bcm:2625]: Good to remove this block as well - https://github.com/sonic-net/sonic-buildimage/blob/61081e89a9ec80095222452574eaacf6498c2270/device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/th5-a7060x6-64pe.config.bcm#L2728-L2743

**Comment by r12f** [comment]
Hi @kperumalbfn , do we still need this change? Looks like we are going to stick on the 10 UC queue now.

**Comment by kperumalbfn** [comment]
@r12f According to Broadcom, this bcm config file supports only 8 UCs. So, we can modify once we get full 10 UC queue support.

**Comment by r12f** [comment]
I see. That sounds good!

**Comment by r12f** [comment]
Basically, we will update all the bcm configs again once we get the 10 UC Q supports, am I right? @kperumalbfn

**Review by kperumalbfn — APPROVED** [review]

**Review by r12f — APPROVED** [review]

**Pull request merged**